scripts/runFSharpLint.fsx

```diff
@@ -142,15 +144,33 @@ Fsdk
     .UnwrapDefault()
 |> ignore<string>
 
-Fsdk
-    .Process
-    .Execute(
+let RunFSharpLint(target: string) =
```

Comment:
@webwarrior-ws please avoid PrimitiveObsession

Reply:
Changed arg's type to FileInfo.